### PR TITLE
Code review request

### PR DESCRIPTION
**Pull request by abdellani**

### CODE REVIEW

**Comment by jofer86** [comment]
Hello!! :wave: :wave: 
   Great effort on this project, the site looks great, the README.MD file was done correctly and has all the necessary items. However, before we can give you :+1: you need to make a few adjustments on the main site.

   1.-When creating a site remember that is very important to use semantic HTML markup, to be more specific, in this case, the site is made out of three distinct components the `<header>` `<main>` and `<footer>` sections, however when looking at your HTML markup I can see that you have `<nav>` tag which contains the navigation menu, said tag should be contained within a `<header>` tag in order to keep things semantically accurate. Also, I can see that you have a `<div>` tag with an id of "container" which contains another `<div>` with the class of "main", to keep the site going with the best practices please do make a `<main>` tag that encapsulates the main content of the site (change de `<div>` tag with the id of "container" for a `<main>` tag that encapsulates everything in that section.

   2.- The navigation bar should mimic the functionality of the original site, meaning that inside the `<li>` tags there should be an anchor tag, with the destination set as '#', just to mimic the functionality. I noticed from your code that you forget to close your <li> tags, please don't forget to close every tag you open, unless is a self-closing tag.
 
   3.- The last item in the navigation bar also has an unnecessary `<div>` tag containing the input of type text with the magnifying glass as a placeholder, remove that div and leave the input tag on its own.

   4.- As far as the main section of the site goes, there should be 3 links present in the main section. The Text that you encapsulate within `<div>` tags should be an `<a>` tag with the destination set to '#', that anchor tag should contain within it a `<div>` tag with a background image (Hint: that image should be  transparent so that you can still see the background). The text on the original site is actually an image.

  5.- As far as the footer goes, the footer is divided into two parts, the part that contains the images, and the part that contains the copyright information and the links, all of that section should be wrapped on a `footer` tag to keep things semantically correct, the four images should be made of four `<a>` tags containing `<div>` tags within it, with the corresponding background images, which should be set with the 'no-repeat' property. The bottom part as I can see from your code is alright, although you forgot to close every `<li>` tag in your list. 

   6.- As a general fix I'm going to ask to see what happens when you resize the screen on your project, we need to mimic the same functionality as the original site, meaning that we need to set max-width to certain elements of the site so that when we resize the page we keep the aesthetics of the design, keep the images centered and with a set max-width to keep our project as close to the original as possible.

   That would be it, once your changes are done, please ask for a new code review.

   Cheers :wave: :wave: 
         Jorge F.

**Comment by abdellani** [comment]
Hi @jofer86 
Thank you very much for the review.
I just want to precise that the <li> tag doesn't require to be closed (https://stackoverflow.com/questions/20550788/does-the-li-tag-in-html-have-an-ending-tag).

Kind regards.
ABDELLANI

**Comment by mcrd25** [comment]
> Hi @jofer86
> Thank you very much for the review.
> I just want to precise that the `<li>` tag doesn't require to be closed (https://stackoverflow.com/questions/20550788/does-the-li-tag-in-html-have-an-ending-tag).
> 
> Kind regards.
> ABDELLANI

Hi @abdellani 👋,

It is still best practice that you close the `<li>` tag!

- Maya

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+main{
+  padding:0;
+  margin-top:0;
+  min-width: 1024px;
+ }
+#main{
+  box-sizing: border-box;
+  background:black;
+  height: 538px;
+  background-image: url(../images/main_bg.jpg);
```
Comment (mcrd25) [inline css/style.css]: Tip 💡: 

I noticed you used `background-image`, `background-repeat` and `background-position` properties a lot. You can shorten this into `background` property.

See [this link](https://developer.mozilla.org/en-US/docs/Web/CSS/background) for more info.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        <li><a href="#"><img src="images/apple-logo.svg" alt="apple logo"></a>
+        <li><a href="#">Store</a>
+        <li><a href="#">Mac</a>
+        <li><a href="#">iPod</a>
+        <li><a href="#">iPhone</a>
+        <li><a href="#">iPad</a>
+        <li><a href="#">iTunes</a>
+        <li><a href="#">Support</a>
+        <li>
+          <input type="text" placeholder="🔍">
```
Comment (mcrd25) [inline index.html]: Hmmm 🤔 maybe there is a better way of adding the search icon, the placeholder is not best way.
Maybe you can address this in the future and create an issue out of it!

Comment (mcrd25) [reply index.html]: Add `autofocus` attribute

**Comment by abdellani** [comment]
Hi @mcrd25 
Thank you for the review. 
Could you please give me more details about the last element in the check list
```
Using display flex for positioning links is recommended (not necessary)
```
Kind regards.

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,178 @@
+nav {
```
Comment (bolah2009) [inline css/style.css]: `SUGGESTION`

Consider adding `margin: 0` for the body element to prevent the default margin on the page.

**Pull request merged**